sql/sql_yacc.yy

```diff
@@ -13933,7 +13909,7 @@ purge_options:
         | CHANGED_PAGE_BITMAPS_SYM BEFORE_SYM real_ulonglong_num
           {
             LEX *lex= Lex;
-            lex->purge_value_list.empty();
+            lex->purge_value_list.clear();
```

Comment:
Again, this is a good one. Exactly why `empty()` method was declared `[[nodiscard]]` in c++17